skll/experiments.py

```diff
@@ -15,9 +15,12 @@
 import math
 import sys
 
+import matplotlib
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import ruamel.yaml as yaml
+import seaborn as sns
 
 from collections import defaultdict
 from io import open
```

Comment:
do we need this any more?

Reply:
No, we don't. Good catch!

Reply:
It's in a whole bunch of other files, I will fix those too.

Reply:
Yes. 
I saw these when grepping files under `skll`.
```
./config.py:16:from io import open
./utilities/summarize_results.py:14:from io import open
./experiments.py:26:from io import open
./data/readers.py:41:from io import open, StringIO
./data/writers.py:17:from io import open
```